CSS/style.css

```diff
@@ -631,3 +714,303 @@ section.contact_me_mobile > hr {
+  display: flex;
+  border-radius: 24px;
+  justify-self: center;
+  margin-bottom: 5rem;
+  margin-top: 5rem;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  height: 496px;
+  gap: 1rem;
+}
+
+.desk_main_content {
+  margin-left: 10px;
+  margin-right: 10px;
+  margin-top: 1rem;
+}
+
+.desk_main_content h1 {
+  font-size: 40px;
+  font-weight: bold;
+  font-family: 'Poppins', system-ui, Courier, monospace;
+  color: #172b4d;
+  text-align: left;
+  margin-top: 1rem;
+}
+
+.desktop-section .desk_main_content p {
+  display: flex;
+  align-items: center;
+  font-size: 16px;
+  font-style: normal;
+  font-family: 'Poppins', sans-serif, Courier, monospace;
+  width: 496px;
+  height: 50px;
+  color: #344563;
+  text-align: left;
+  margin-top: 2rem;
+  margin-left: 1px;
+}
+
+.desk_main_img1 {
+  display: flex;
+  background-image: url('../Images/deskcard1.svg');
+  background-repeat: no-repeat;
+  margin-top: 10px;
+  width: 544px;
+  height: 448px;
+  border-radius: 8px;
+  margin-right: 10px;
+}
+
+.desk_main_img2 {
+  display: flex;
+  background-image: url('../Images/deskcard1.svg');
+  background-repeat: no-repeat;
+  margin-top: 10px;
+  width: 544px;
+  height: 448px;
+  border-radius: 8px;
+  margin-right: 10px;
+}
+
+.desk_main_img3 {
+  display: flex;
+  background-image: url('../Images/deskcard1.svg');
+  background-repeat: no-repeat;
+  margin-top: 10px;
+  width: 544px;
+  height: 448px;
+  border-radius: 8px;
+  margin-right: 10px;
+}
+
+.desk_main_img4 {
+  display: flex;
+  background-image: url('../Images/deskcard1.svg');
+  background-repeat: no-repeat;
+  margin-top: 10px;
+  width: 544px;
+  height: 448px;
+  border-radius: 8px;
+  margin-right: 10px;
+}
+
+section.desktop_ams {
+  display: flex;
+  width: 100%;
+  border: 1px solid #dfe1e6;
+  background-color: #fff;
+  padding-top: 5rem;
+  padding-bottom: 10rem;
+  padding-left: 5rem;
+  border-top-right-radius: 150px;
+}
+
+section.desktop_ams .about_part1 {
+  width: 40%;
+}
+
+section.desktop_ams .about_part2 {
+  width: 60%;
+}
+
+section.desktop_ams .about_part1 h2 {
+  display: flex;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  font-family: 'Poppins', system-ui, Courier, monospace;
+  color: #091e42;
+  text-align: left;
+  margin-top: 10px;
+}
+
+section.desktop_ams .about_part1 p {
+  display: flex;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  font-family: 'Poppins', system-ui, Courier, monospace;
+  width: 447px;
+  height: 114px;
+  margin-left: 1px;
+  color: #344563;
+  text-align: left;
+  margin-top: 10px;
+}
+
+section.desktop_ams .about_part2 ul {
+  list-style: none;
+}
+
+section.desktop_ams .about_part2 ul li ul {
+  display: flex;
+  list-style: none;
+}
+
+section.desktop_ams .about_part2 ul li ul li {
+  display: flex;
+  list-style: none;
+  width: 140px;
+  height: 120px;
+}
+
+.contact_me_desktop {
```

Comment:
- **[OPTIONAL]** Please consider removing the bottom spacing from the contact section 👌🏼 

### Your Design
![image](https://user-images.githubusercontent.com/68030297/229246223-fe111dd7-b63d-4e48-98d5-bbd45e393039.png)

### Figma Design
![Screenshot from 2023-03-31 23-51-32](https://user-images.githubusercontent.com/68030297/229246257-67054527-adde-418a-bb35-b89414e759e6.png)